stylus-sdk/src/call/mod.rs

```diff
@@ -14,7 +14,10 @@
 use alloc::vec::Vec;
 use alloy_primitives::Address;
 
-pub use self::{context::Call, error::Error, raw::RawCall, traits::*, transfer::transfer_eth};
+pub use self::{
+    context::Call, error::EncodeError, error::Error, raw::RawCall, traits::*,
+    transfer::transfer_eth,
+};
```

Comment:
I don't think you need to import `EncodeError`. If you do, we should tweak the impl

Reply:
It needs to be visible outside the crate so it can be called in expansions of the `SolidityError` derive macro. How do you want to tweak the impl?

Reply:
I think deleting the import just works out of the box. We're using the trait directly with a fully qualified path in the macro so I think it should just connect

Reply:
Oh wait I see what happened, I meant to comment on the ERC 20 example

Reply:
Ok looks good, my bad fam :)